internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,79 @@
 package daysteps
 
 import (
-	...
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint5-final/internal/personaldata"
+	"github.com/Yandex-Practicum/go1fl-sprint5-final/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
 // создайте структуру DaySteps
-...
-
+type DaySteps struct {
+	Steps int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) (err error) {
+
+	s := strings.Split(datastring, ",")
+
+	if len(s) == 2 {
```

Comment:
Лучше инвертировать условие
```suggestion
	if len(s) != 2 {
		err := errors.New("Invalid arguments count")
		return err
	}
	....
```